### PR TITLE
Use master branch for main development

### PR DESCRIPTION
**Pull request by physikerwelt**
as discussed on IRC with @d00rman

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +18,10 @@
         "url": "git://github.com/mathjax/MathJax-node.git"
     },
     "dependencies": {
-       "jsdom": "*",
-       "speech-rule-engine": "*",
-       "yargs": "*"
+        "jsdom": "3.1.1",
+        "speech-rule-engine": "*",
+        "yargs": "*",
+        "MathJax": "git+https://github.com/wikimedia/MathJax#mathoid-0-2-8"
```
Comment (d00rman) [inline package.json]: Since we are now merging into master both repos, IMO this should point to the master branch

Comment (d00rman) [inline package.json]: Hm, in fact, that should be done separately, not in the `mathoid-0-2-8` branch in order not to break the deps.

**Comment by physikerwelt** [comment]
Yes. I know that but I don't know to that in one step on github
"Add more commits by pushing to the mathoid-0-2-8 branch on wikimedia/MathJax-node."
Maybe I can just push to the master branch?

**Comment by d00rman** [comment]
> Maybe I can just push to the master branch?

There are basically two ways:
1. branch off of `mathoid-0-2-8`, commit there and then create a PR
2. keep this PR and open another one with just that change

IMHO, option (1) is better as it makes the PR self-contained

**Comment by physikerwelt** [comment]
I followed (1).

**Comment by physikerwelt** [comment]
See the diff here https://github.com/wikimedia/MathJax-node/compare/mathoid-0-2-8...wikimedia:master

**Pull request merged**